src/Components/Form.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+
+function Form (props) {
+    const [input, setInput] = useState({name: '', email: '', role: ''})
+
+    function handleChange(event) {
+      setInput({ ...input, [event.target.name]: event.target.value });
+      // console.log(event.target.value)
+
+    }
+
+    function submitForm(event) {
+      event.preventDefault();
+            const newMember = {
+                  ...input,
+              };
+      props.addMember(newMember);
+      setInput({Name: "", Email: "", Role: "" })
+
```

Comment:
This isn't clearing the input like it should because you are using capitalized names for your object keys here